cob_people_detection/ros/launch/people_detection.launch

```diff
@@ -18,36 +22,69 @@ for Care-O-bot default is cam3d_nodelet_manager -->
   <arg name="launch_detection_tracker" default="true"/>
   <arg name="launch_face_capture" default="true"/>
   <arg name="launch_coordinator" default="true"/>
-  <arg name="display_results_with_image_view" default="true"/>		<!-- set to false if you do not like to display the camera image with names attached to detected faces within an image_view window --> 
+  <arg name="display_results_with_image_view" default="true"/>		<!-- set to false if you do not like to display the camera image with names attached to detected faces within an image_view window -->
 
   <!-- nodelet or non-nodelet specific launch files -->
-  <group unless="$(arg using_nodelets)">
-    <!-- launch files for usage without nodelets -->
-    <include file="$(find cob_people_detection)/ros/launch/sensor_message_gateway.launch">
-      <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
-      <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
-    </include>
+   <group unless="$(arg using_nodelets)">
+     <!-- launch files for usage without nodelets -->
+     <include file="$(find cob_people_detection)/ros/launch/sensor_message_gateway.launch">
+       <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
+       <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
+     </include>
+   </group>
+   <group if="$(arg using_nodelets)">
+     <!-- special launch files for usage with nodelets -->
+     <node if="$(arg start_manager)" pkg="nodelet" type="nodelet" name="$(arg nodelet_manager)"  args="manager" output="screen"/>
+     <include file="$(find cob_people_detection)/ros/launch/sensor_message_gateway_nodelet.launch">
+       <arg name="nodelet_namespace" value="$(arg camera_namespace)"/>
+       <arg name="nodelet_manager" value="$(arg nodelet_manager)"/>
+       <arg name="colorimage_in_topic" value="$(arg colorimage_in_topic)"/>
+       <arg name="pointcloud_rgb_in_topic" value="$(arg pointcloud_rgb_in_topic)"/>
+     </include>
+   </group>
+
+  <!-- remaining launch files -->
+  <param name="data_storage_directory" type="string" value="$(env HOME)/.ros/cob_people_detection/files/"/>
+  <param name="face_detector_method" type="int" value="$(arg face_detector_method)"/> <!-- 0 -> hog: Histogram of Oriented Gradients,
```

Comment:
Do you really need to create this parameter here? Is it used from some program later on?